ethers-core/src/types/chain.rs

```diff
@@ -460,6 +463,10 @@ impl Chain {
 
             Boba => ("https://api.bobascan.com/api", "https://bobascan.com"),
 
+            ZKSync => {
```

Comment:
one last thing, this wasn't renamed

```suggestion
            Zksync => {
```

Reply:
resolved in another commit :D thx